### DISCUSSION TITLE
Work with Azure Repositories

### DISCUSSION THREAD

**Discussion by ADCairex** [rdilweb/docusaurus-plugin-remote-content#35] [Q&A]
I'm trying to use this plugin with an azure repository but not works, is because i'm doing something wrong or its because it can't be used with it?

**Comment by RDIL** [comment]
Hey,
It should be possible, but you would need to use a different base URL from the one in the readme's example.  You also may need to implement authentication through the `requestOptions` variable.  Beyond that, I can't help, since you haven't provided a reproduction or any more context.

**Comment by ADCairex** [comment]
I think the probles is that the markdown file is gived by a queryString not in the baseUrl and I put the auth token to acces in the axios options and return me a 200 OK code.

**Reply by ADCairex** [reply]
https://dev.azure.com/Organization/_git/Test?path=/README.md

**Reply by RDIL** [reply]
Hmm, you could maybe hack around it by setting the baseUrl to `https://dev.azure.com/Organization/_git/Test?path=` with the files option being `["", "", ""]` and then use the request config to modify it or something.
